### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 12th of September 2023 17:30 CEST / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#1175] [Flatcar Office Hours]
- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording:  https://www.youtube.com/watch?v=QbT45-yxOEo
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

## Agenda items

- The Image-Based Linux Summit says "Hi"! We'll dial in the conference room where the summit is taking place today. @bluca will give a brief intro + update on the summit.
- @tormath1 - recap - Flatcar team has organized a bug smashing day the 8th Sept.: the concept is to go through the list of GitHub issues to see which one is still relevant / need more information / miss labels / etc. https://hackmd.io/vguI7WOXSJuTNqZ1D8KBuA
We closed around 16 issues and created 4 PRs for updating the documentation. Thanks to everyone involved!

### Release Planning
- https://github.com/orgs/flatcar/projects/7/views/8

### Administrative tasks before meeting starts
- [ ] Create a comment with the list of closed/opened issues/PRs (hint: also sort by time updated)
### Administrative tasks after meeting concluded
- [ ] Create a new Github discussion for the next Flatcar Office Hours: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-office-hours
- [ ] Close this discussion as "Resolved".

**Comment by miao0miao** [comment]
## PR's Closed
* [#1170](https://github.com/flatcar/Flatcar/pull/1170):  README: Link Flatcar Mastodon account
* [#1167](https://github.com/flatcar/Flatcar/pull/1167):  Rename adopters.md to ADOPTERS.md
* [#1116](https://github.com/flatcar/Flatcar/pull/1116):  deleted archived repos from maintainers list

## PR's Opened
* [#1170](https://github.com/flatcar/Flatcar/pull/1170):  README: Link Flatcar Mastodon account
* [#1167](https://github.com/flatcar/Flatcar/pull/1167):  Rename adopters.md to ADOPTERS.md

## Issues Opened
* [#1174](https://github.com/flatcar/Flatcar/issues/1174):  update: go
* [#1173](https://github.com/flatcar/Flatcar/issues/1173):  update: python
* [#1172](https://github.com/flatcar/Flatcar/issues/1172):  Run Flatcar update test in GitHub Action PR CI
* [#1171](https://github.com/flatcar/Flatcar/issues/1171):  systemd-tmpfiles.service complains about some duplicate config lines.
* [#1169](https://github.com/flatcar/Flatcar/issues/1169):  Correct RemainsAfterExit=true to be RemainAfterExit=true in nvidia.service
* [#1166](https://github.com/flatcar/Flatcar/issues/1166):  Release Flatcar Container Linux Alpha 3717.0.0, Beta 3602.1.5, Stable 3510.2.7, LTS 3033.3.17
* [#1164](https://github.com/flatcar/Flatcar/issues/1164):  update: open-vm-tools
* [#1163](https://github.com/flatcar/Flatcar/issues/1163):  Flatcar fails to install current (GPG signature check failed)
* [#1162](https://github.com/flatcar/Flatcar/issues/1162):  Unable to install nvidia-drivers on Flatcar Container Linux by Kinvolk stable 3033.2.2
* [#1161](https://github.com/flatcar/Flatcar/issues/1161):  selinux: Docs state it works, but issue tracker does not corroborate

## Issues Closed
* [#1173](https://github.com/flatcar/Flatcar/issues/1173):  update: python
* [#1164](https://github.com/flatcar/Flatcar/issues/1164):  update: open-vm-tools
* [#1163](https://github.com/flatcar/Flatcar/issues/1163):  Flatcar fails to install current (GPG signature check failed)
* [#1162](https://github.com/flatcar/Flatcar/issues/1162):  Unable to install nvidia-drivers on Flatcar Container Linux by Kinvolk stable 3033.2.2
* [#1156](https://github.com/flatcar/Flatcar/issues/1156):  update: Kernel (AMD Inception)
* [#1155](https://github.com/flatcar/Flatcar/issues/1155):  update: intel-microcode and Kernel (downfall etc.)
* [#1151](https://github.com/flatcar/Flatcar/issues/1151):  Error: update failed
* [#1137](https://github.com/flatcar/Flatcar/issues/1137):  [Docs] Consider updating references to Apache container
* [#1129](https://github.com/flatcar/Flatcar/issues/1129):  Add docs for systemd-sysupdate
* [#1128](https://github.com/flatcar/Flatcar/issues/1128):  Publish built images in sysext-bakery with GitHub Action as GitHub releases for systemd-sysupdate
* [#1099](https://github.com/flatcar/Flatcar/issues/1099):  update: grub
* [#1087](https://github.com/flatcar/Flatcar/issues/1087):  update: qemu
* [#1063](https://github.com/flatcar/Flatcar/issues/1063):  If I want to compile something to run on flatcar, what is the most appropriate linux environment to do so?
* [#1041](https://github.com/flatcar/Flatcar/issues/1041):  Release Flatcar Container Linux Alpha 3619.0.0, Beta 3602.1.0, Stable 3510.2.2, LTS 3033.3.13
* [#1034](https://github.com/flatcar/Flatcar/issues/1034):  NVIDIA GPUs: add documentation on how to use (for users), and pointers to ebuilds (for devs)
* [#984](https://github.com/flatcar/Flatcar/issues/984):  Make upstream systemd-sysext reload the units by default
* [#887](https://github.com/flatcar/Flatcar/issues/887):  AWS AMI catalog does not show later version of Flatcar LTS in its catalog page
* [#840](https://github.com/flatcar/Flatcar/issues/840):  Intermittent AWS AMI corruptions
* [#768](https://github.com/flatcar/Flatcar/issues/768):  Support systemd-sysext OEM update payloads in update-engine
* [#767](https://github.com/flatcar/Flatcar/issues/767):  Make flatcar-update provide OEM payloads
* [#685](https://github.com/flatcar/Flatcar/issues/685):  Systemd-networkd route setup leads to failed interface state (v249 but also in v250 now)
* [#683](https://github.com/flatcar/Flatcar/issues/683):  NTP server is giving a time far in the future
* [#609](https://github.com/flatcar/Flatcar/issues/609):  Has the GCP image for 2765.2.6 been deleted or the required permissions changed?
* [#588](https://github.com/flatcar/Flatcar/issues/588):  flatcar-install not working on some distros ?
* [#585](https://github.com/flatcar/Flatcar/issues/585):  [Tracking] AWS Cloudwatch metrics reporting issue with CGroupsV2
* [#563](https://github.com/flatcar/Flatcar/issues/563):  containerd-shim processes are leaking inotify instances with cgroups v2
* [#505](https://github.com/flatcar/Flatcar/issues/505):  systemd ratelimit being hit after upgrade
* [#402](https://github.com/flatcar/Flatcar/issues/402):  [RFE] Add instruction for reporting security issues to README.md and via GitHub security policy

**Comment by tormath1** [comment]
Flatcar team has organized a bug smashing day the 8th Sept.: the concept is to go through the list of GitHub issues to see which one is still relevant / need more information / miss labels / etc.

We closed around 16 issues and created 4 PRs for updating the documentation. Thanks to everyone involved! 
https://hackmd.io/vguI7WOXSJuTNqZ1D8KBuA
